api/features/serializers.py

```diff
@@ -244,6 +256,87 @@ class Meta:
     identity = _IdentitySerializer()
 
 
+class FeatureStateValueEdgeIdentityField(serializers.Field):
+    def to_representation(self, obj):
+        identity_id = self.parent._get_identity_uuid()
+        return obj.get_value(identity_id=identity_id)
+
+    def get_attribute(self, instance):
+        return instance
+
+    def to_internal_value(self, data):
+        feature_state_value_dict = FeatureState().generate_feature_state_value_data(
+            data
+        )
+
+        data = {**feature_state_value_dict}
+        fs_value_serializer = FeatureStateValueSerializer(data=data)
+        fs_value_serializer.is_valid(raise_exception=True)
+        return FeatureStateValue(**data).value
+
+
+class EdgeFeatureField(serializers.Field):
+    def to_representation(self, obj):
+        return obj.id
+
+    def to_internal_value(self, data):
+        feature = Feature.objects.get(id=data)
+        return feature
+
+
+class FeatureStateSerializerWithEdgeIdentity(serializers.Serializer):
```

Comment:
Should this be `EdgeIdentityFeatureStateSerializer` instead? I'm not sure it's _with_ an edge identity is it?